### ISSUE TITLE
typing delayed in radian console

### ISSUE THREAD

**Issue by ddzhang3** [randy3k/radian#262]
My radian console seems to have 2 uncomfortable issues: 
1. It starts slowly;
2. there's a delay in the second character in every word-typing,  like this:
![radian](https://user-images.githubusercontent.com/70323160/108180309-9023a480-7141-11eb-90dc-7f5135920ac3.gif)

Instead, the original R console does neither starts slowly nor suffers from a delay:
![R](https://user-images.githubusercontent.com/70323160/108180891-38396d80-7142-11eb-96ff-e2c0d1ff0c46.gif)

> >r --version
radian version: 0.5.10
r executable: /home/apps/R/4.0.2/lib64/R/bin/R
r version: 4.0.2
python executable: /home/users/ddzhang/python/3.9.1-env/bin/python3
python version: 3.9.1

How to avoid this two issues? Thanks

**Comment by randy3k** [comment]
To be honest, I don't know exactly what causes it. Maybe trying to disable 'auto completion while typing'?

**Comment by MilesMcBain** [comment]
Thanks for making these videos @ddzhang3. My colleague @anthonynorth and I have also been seeing this for a while.

I found that disabling auto completion while typing does improve things somewhat. However it still feels slightly sluggish compared to typing in a vscode editor window, which is a bit surprising to me since I think vscode is probably doing way more work on the fly.

**Comment by ddzhang3** [comment]
@MilesMcBain Yes, it seems to work without obvious pause after I start the radian console for a while, maybe half hour later. However, the pause appears again in the next fresh start.

**Comment by Fred-Wu** [comment]
I found this a bit annoying as well. 
Also it becomes more sluggish after finishing a big job.

**Comment by karchern** [comment]
are there any updated on this? I'm seeing this too and the problem is so bad that I can't really use radian, which is unfortuante since it's otherwise so superiour to the standard R console

**Comment by nickhir** [comment]
I am facing similar issues.. does anyone have suggestions?
